app/assets/stylesheets/application.scss

```diff
@@ -1181,52 +1211,42 @@ div.field {
   margin-left: 25px;
 }
 
-#return-to-log-in {
-  margin-top: 25px;
-  margin-left: 40.2%;
-}
-
-.reset-password-button {
-  margin-left: 12%;
-  margin-top: 10px;
-}
-
-#reset-password-button {
-  border: 0;
-  background: none;
-}
-
 .reset-password-form h2 {
-  margin-left: 4%;
+  font-family: 'Architects Daughter', courier;
+  margin-left: 13.5%;
```

Comment:
Properties should be ordered font-family, margin-bottom, margin-left, margin-right